### PR TITLE
Initial version on data fetching and config for Redux reducers

### PR DESCRIPTION
**Pull request by DLMR**
* Also removes old data fetching logic, will add back some of it in the future.
* Adds a more complex example.

### CODE REVIEW

File: src/app/shared/flux/create-store.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ import { routeReducer } from 'redux-simple-router';
  * the reducers. The callback itself will take a function as the parameter that in turn takes the reducers to update.
  */
 export function createReduxStore(extraReducers, ...extraMiddlewares) {
-    return (callback) => {
-        return (initialState) => {
+    return (callback) =>
+        (initialState) => {
```
Comment (andreasrs) [inline src/app/shared/flux/create-store.js]: Any particular reason for this change? In general we prefer keeping curlybrakets for clarity

**Comment by andreasrs** [comment]
It's a good start! We can merge this. I'll improve the "complex" example a bit in a separate PR.

File: src/app/shared/flux/create-store.js (hunk 1 above)
Comment (DLMR) [inline src/app/shared/flux/create-store.js]: It was not needed, it is a single line so return will be implicit. Makes it cleaner and easier to read in my opinion but I'm fine with changing it back if you disagree.

Comment (andreasrs) [inline src/app/shared/flux/create-store.js]: We can leave this as is.

**Pull request merged**